kolibri/plugins/coach/assets/src/constants/index.js

```diff
@@ -40,6 +40,30 @@ export const PageNames = {
   LESSON_SELECT_RESOURCES: 'LESSON_SELECT_RESOURCES',
   LESSON_PREVIEW_SELECTED_RESOURCES: 'LESSON_PREVIEW_SELECTED_RESOURCES',
   LESSON_PREVIEW_RESOURCE: 'LESSON_PREVIEW_RESOURCE',
+
+  /* Lessons */
+  LESSON_CREATION_ROOT: 'LESSON_CREATION_ROOT',
```

Comment:
I havent found value in having these lesson page names in a different object. It rather adds complexity as we have to import more objects, and we dont inmidetaly notice if there is already a duplicated page name (It happened to me with the "SUMMARY" page names for lesson summary which I will refactor to be calld LESSON_SUMMARY, but just to give an example).

Reply:
I think the deduplication piece is a very strong argument in favour of a single PageNames object here.